src/Api.ts

```diff
@@ -306,6 +320,37 @@ class Api {
     });
   }
 
+  /**
+   * Gets messages from the specified channel.
+   *
+   * @param limit The max number of messages to return (1-100)
+   * @param channelId The ID of the channel
+   * @param filter Object to filter messages
+   */
+  getMessages(
+    channelId: string,
+    limit = 50,
+    filter: FilterType = undefined,
```

Comment:
Is it necessary to specify "undefined" here versus just `filter: FilterType,`?

Reply:
Specifying `= undefined` gives it a default value so it's not a mandatory parameter.

Reply:
Yes yes yes, my bad :)